packages/protocol/test/governance/network/governance.ts

```diff
@@ -1557,6 +1557,12 @@ contract('Governance', (accounts: string[]) => {
       })
     })
 
+    it('should not revert when proposals are not in the Referendum stage', async () => {
+      await governance.vote(proposalId, index, VoteValue.Yes)
+      await timeTravel(referendumStageDuration, web3)
+      await governance.revokeVotes()
```

Comment:
I think this test case deletes the proposal inside the `requireDequeuedAndDeleteExpired`
I wonder if we can also test when there is a proposal in the dequeue in `Approval` that hasn't expired yet